README.md

```diff
@@ -141,6 +142,23 @@ Sentry as part of the extra data, if it was the last action before an error.
 
 This option was introduced in version 1.1.1.
 
+#### `getUserContext` _(Optional Function)_
+
+Signature: `state => userContext`
+
+The [user context] is an important part of the error reporting. When
+`getUserContext` is a truthy function, the result of `getUserContext`
+will set the the [user context] before sending the error report.
```

Comment:
I think it’s standard to only make the first reference to a given term a link.

Reply:
:+1: